scripts/globals/quests.lua

```diff
@@ -31,43 +31,63 @@ local check_enum =
     onZoneIn = 6
 }
 
+local function validateQuest(entity, quest, varname, val, caller, get)
+    local success = true
+    local prefix = "[Quest Parameter Error] "
+
+    if entity == nil then
+        print(prefix .. "entity cannot be nil")
+        success = false
+    elseif quest == nil then
+        print(prefix .. "quest cannot be nil")
+        success = false
+    elseif varname == nil or varname == '' then
+        print(prefix .. "varname cannot be nil or empty")
+        success = false
+    end
+    -- todo what is caller?
```

Comment:
caller was supposed to just be a debug string for any errors, just i forgot to rename it

Reply:
though actually it can probably fuck off